index.js

```diff
@@ -75,10 +75,11 @@ class SideMenu extends Component {
   }
 
   componentDidMount() {
-    this.setState({
-      shouldRenderMenu: true,
-    });
-
+    setTimeout(() => {
+      this.setState({
+        shouldRenderMenu: true,
+      });
+    }, 500);
```

Comment:
Unfortunately, it doesn't work. I've tried using `InteractionManager` for this purpose - no luck :(